.github/workflows/gh-pages-deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           npm install
           npm run lint
           npm test
-          npm run build
```

Comment:
Why has the build task been renamed to webv?

Reply:
Not rename. Separate task. Uses the `webpack.web.js` for build